api/newrelic.ini

```diff
@@ -0,0 +1,270 @@
+#
+error_collector.ignore_status_codes = 401 402 403 404 405 406 415 503
+
+# Browser monitoring is the Real User Monitoring feature of the UI.
+# For those Python web frameworks that are supported, this
+# setting enables the auto-insertion of the browser monitoring
+# JavaScript fragments.
+browser_monitoring.auto_instrument = false
+
+# A thread profiling session can be scheduled via the UI when
+# this option is enabled. The thread profiler will periodically
+# capture a snapshot of the call stack for each active thread in
+# the application to construct a statistically representative
+# call tree. For more details on the thread profiler tool, see
+# https://docs.newrelic.com/docs/apm/apm-ui-pages/events/thread-profiler-tool/
+thread_profiler.enabled = true
+
+# Your application deployments can be recorded through the
+# New Relic REST API. To use this feature provide your API key
+# below then use the `newrelic-admin record-deploy` command.
+# api_key =
+
+# Distributed tracing lets you see the path that a request takes
+# through your distributed system. For more information, please
+# consult our distributed tracing planning guide.
+# https://docs.newrelic.com/docs/transition-guide-distributed-tracing
+distributed_tracing.enabled = true
+
+# When storing errors in database, distributed tracing solution captures the database query
+# and sends the full, unscrubbed message to New Relic. This enablement will ensure that
+# no PII data is captured in messages of new relic.
+strip_exception_messages.enabled = true
+
+# ---------------------------------------------------------------------------
+
+#
+# The application environments. These are specific settings which
+# override the common environment settings. The settings related to a
+# specific environment will be used when the environment argument to the
+# newrelic.agent.initialize() function has been defined to be either
+# "local", "stage", "performance", "training", "prod", or "uat".
+#
+
+[newrelic:local] 
+# Don't turn on data reporting by default when running the API locally.
+#
+# To enable New Relic locally, set the following variables:
+# - developer mode: false
+# - monitor_mode: true
+# - license_key: retrieved from New Relic here: https://one.newrelic.com/launcher/api-keys-ui.launcher
+#
+# NOTE: DO NOT COMMIT THE LICENSE KEY IN A GIT COMMIT.
+#
+# Less scary note: do not wrap your license key in quotes, it should look like this:
+#   license_key=1234abcd
+#
+app_name = SIMPLER-GRANTS-API-LOCAL
+developer_mode = true
+monitor_mode = false
+license_key=replace_me
+
+application_logging.enabled = false
+application_logging.forwarding.enabled = false
+application_logging.local_decorating.enabled = false
+
+[newrelic:staging]
+app_name = SIMPLER-GRANTS-API-STAGING
+monitor_mode = true
+
+[newrelic:prod]
+app_name = SIMPLER-GRANTS-API-PROD
+monitor_mode = true
+
+[newrelic:dev]
+app_name = SIMPLER-GRANTS-API-DEV
+monitor_mode = true
```

Comment:
@coilysiren - I think we talked about this a few weeks ago, but what landing scheme did we land on for the API app in New Relic?

Reply:
this looks fine to me - the `SIMPLER-GRANTS-` bit is probably superfluous but if that matches the frontend then I'm down to keep it